pkg/cluster/repository/ClusterRepository.go

```diff
@@ -48,13 +49,14 @@ type ClusterRepository interface {
 	FindOneActive(clusterName string) (*Cluster, error)
 	FindAll() ([]Cluster, error)
 	FindAllActive() ([]Cluster, error)
-
 	FindById(id int) (*Cluster, error)
 	FindByIds(id []int) ([]Cluster, error)
 	Update(model *Cluster) error
 	Delete(model *Cluster) error
 	MarkClusterDeleted(model *Cluster) error
 	UpdateClusterConnectionStatus(clusterId int, errorInConnecting string) error
+	FindActiveClusters() ([]Cluster, error)
```

Comment:
change the method name to findAllActiveClusterNames and change the return type to []string,since u only need cluster names.

Reply:
Now I need more informations of the cluster like cluster id to update the clusters